src/lib/modules/whisper/js/communicationFunctions.js

```diff
@@ -93,8 +93,7 @@ function listenTo(options, callback) {
         return callback(null, data);
       }
       promise.cb(payload, data, result);
-    })
-    .catch(callback);
```

Comment:
After making all of the web3 beta.34 modules direct dependencies of embark, the object returned by `subscribe(...).on(...)` doesn't have a `.catch` method. With the invocation removed, everything seems to be working okay in `embark_demo`.